packages/xo-web/src/xo-app/backup/overview/tab-jobs.js

```diff
@@ -49,7 +51,13 @@ const Li = props => (
   />
 )
 
+const isMirrorBackup = item => item?.type === 'mirrorBackup'
```

Comment:
The `modes` column being plural, i'm not suprised to see for the same backup:
`Mirror backup`
`Delta backup`

But for clarity, I can change it. I have no opinion on it.

Reply:
It's plural because you can have different modes running in the same job. But if you see "Mirror Backup" and "Delta Backup", one might think that 2 types are going to run when only one has been selected. So I think it's better to change it and have dedicated types for mirror backups.